__tests__/tools/reporter/NswgReporter.test.js

```diff
@@ -0,0 +1,118 @@
+    username: 'asdfghjkjl',
+    name: 'asdfghjkl',
+    bio: null,
+    website: null,
+    location: null,
+    created_at: '2018-07-04T13:30:25.133Z',
+    url: 'https://hackerone.com/asdfghjkl',
+    anc_triager: false,
+    hackerone_triager: false,
+    profile_picture_urls: { 
+      small: '/assets/avatars/default-12345.png',
+      medium: '/assets/avatars/default-123456.png',
+      xtralarge: '/assets/avatars/default-1234567.png' },
+    disabled: false,
+    report_count: 6,
+    target_count: 1,
+    reputation: 142,
+    rank: 2630,
+    signal: 7,
+    impact: null,
+    signal_percentile: 96,
+    impact_percentile: null,
+    total_mentees: 0,
+    mentor: null } 
+  }
+
+const reporter = new NswgReporter({report})
+let wgReport = {}
+
+describe("Testing NswgReporter", ()=> {
+  test("Testing getTitle", () => {
+    expect(reporter.getTitle()).toBe("[TBD]")
+  })
+  test("Testing getOverview", () => {
+    expect(reporter.getOverview()).toBe("overview")
+  })
+  test("Testing getCreatedAt", () => {
+    expect(reporter.getCreatedAt()).toBe("2018-07-13")
+  })
+  test("Testing getUpdatedAt", () => {
+    expect(reporter.getUpdatedAt()).toBe(null)
+  })
+  test("Testing getPublishAt", () => {
+    expect(reporter.getPublishAt()).toBe("2018-08-22")
+  })
+  test("Testing getAuthorUsername", () => {
+    expect(reporter.getAuthorUsername()).toEqual({"name": "asdfghjkl", "username": "asdfghjkjl", "website": null})
+  })
+  test("Testing getModuleName", () => {
+    expect(reporter.getModuleName()).toBe("[TBD]")
+  })
+  test("Testing getCVEs", () => {
+    expect(reporter.getCVEs()).toBe(null)
+  })
+  test("Testing getVulnerableVerison", () => {
+    expect(reporter.getVulnerableVerison()).toBe(null)
+  })
+  test("Testing getPatchedVersion", () => {
+    expect(reporter.getPatchedVersion()).toBe(null)
+  })
+  test("Testing getRecommendation", () => {
+    expect(reporter.getRecommendation()).toBe("update [TBD] to null or higher")
+  })
+  test("Testing getReferences", () => {
+    expect(reporter.getReferences()).toEqual(["https://hackerone.com/reports/123456"])
+  })
+  test("Testing getCVSS", () => {
+    expect(reporter.getCVSS()).toBe("[TBD]")
+  })
+  test("Testing getCVSSScore", () => {
+    expect(reporter.getCVSSScore()).toBe("[TBD]")
+  })
+  test("Testing get for wgReport", () => {
+    return Promise
+      .resolve().then(() => {
+        return reporter.get()
+      }).then((rep)=> {
+        wgReport = rep
+        expect(wgReport).toEqual({"author": {"name": "asdfghjkl", "username": "asdfghjkjl", "website": null}, "coordinating_vendor": null, "created_at": "2018-07-13", "cves": null, "cvss_score": "[TBD]", "cvss_vector": "[TBD]", "id": 0,"module_name": "[TBD]", "overview": "overview", "patched_versions": null, "publish_date": "2018-08-22", "recommendation": "update [TBD] to null or higher", "references": ["https://hackerone.com/reports/123456"], "title": "[TBD]", "updated_at": null, "vulnerable_versions": null})   
```

Comment:
Consider moving this expectation to an external mock file since its so big?

Reply:
> @waveywaves Great stuff.
> 
> * I think this is a great start for us to merge in and can optimize on later.
> * Left a couple of comments for quick fixes we can push now
> * CI is failing, perhaps you can look and update the tests accordingly? https://travis-ci.com/nodejs/security-wg/builds/95697612

-   "updated_at": null, Expected
   +   "updated_at": "1970-01-01", Received

the above output which is seen after a failed test. The reporter is not parsing the date correctly and hence we get the unix timestamp genesis date as a return for passing null. I am making changes to the reporter regarding this and putting it in another PR. https://github.com/nodejs/security-wg/pull/463